website/content/docs/running-staticcheck/cli/build-tags/index.md

```diff
@@ -0,0 +1,167 @@
+However, in reality, code bases can be complex, and it isn't always clear which sets of build tags make use of what code.
+After all, if we always knew, we wouldn't have dead code to begin with.
+
+Another example involves control flow. Consider this package, again in txtar format:
+
+```go
+-- api_linux.go --
+package pkg
+
+func dieIfUnsupported() { panic("unsupported") }
+-- api_windows.go --
+package pkg
+
+func dieIfUnsupported() {}
+-- shared.go --
+package pkg
+
+func foo() {}
+
+func Entry() {
+	dieIfUnsupported()
+	foo()
+}
+```
+
+Here, `dieIfUnsupported` panics unconditionally on Linux, but not on Windows.
+Because Staticcheck takes control flow into consideration, this means that `foo` is unused on Linux but used on Windows.
+
+Several checks have this sort of false positive, not just U1000.
+
+## Solution
+
+The solution to this problem is to run Staticcheck multiple times with different build tags and to merge the results.
+
+At first glance, one might think that Staticcheck should be able to do this fully automatically: look at all build tags, find all unique combinations, and check them all.
+However, this doesn't scale.
+To be correct, Staticcheck would have to take dependencies and their tags into consideration, too.
+Virtually all code depends on the Go standard library, and the Go standard library supports a plethora of operating systems, architectures, and a number of tags such as `netgo`.
+All in all, there are thousands of unique combinations.
+Checking all of these would take far too long.
+
+However, the number of build configurations you care about is probably much smaller.
+Your software probably support 2-3 operating systems on 1-2 architectures,
+and maybe has a debug and a release build.
+This makes for a lot fewer combinations that need to be checked.
+These are probably the same combinations you're already checking in CI, too, by running their tests.
+This will become useful in a bit.
+
+### The `-merge` flag
+
+Using the `-merge` flag, Staticcheck can merge the results of multiple runs.
+It decides on a per-check basis whether any run or all runs have to have reported an issue for it to be valid.
+It also takes into consideration which files were checked by which run, to reduce false negatives.
+
+In order to use `-merge`, the runs to be merged have to use the `-f binary` flag.
+This outputs results in a binary format containing all information required by `-merge`.
+When using `-merge`, arguments are interpreted as file names instead of import paths, so that `staticcheck -merge file1 file2` will read the files `file1` and `file2`, which must contain the output of `staticcheck -f binary` runs, and merge them.
+
+```terminal
+$ GOOS=linux staticcheck -f binary >file1
+$ GOOS=windows staticcheck -f binary >file2
+$ staticcheck -merge file1 file2
+...
+```
+
+
+Alternatively, if no arguments are passed, `staticcheck` will read from standard input instead.
```

Comment:
I'd say it's better to repeat the flag:

 >  […] `staticcheck -merge` will read […]

To avoid any confusion and accidentally making people think that a naked call to `staticcheck` will do here.